src/main/scala/ostrich/parikh/core/FinalConstraintsSolver/CatraBasedSolver.scala

```diff
@@ -207,26 +208,32 @@ class CatraBasedSolver(private val freshIntTerm2orgin: Map[Term, Term]) extends
       result.setStatus(ProverStatus.Sat)
       return result
     }
+    // for (c <- constraints; aut <- c.auts) {
+    //   aut.toDot("catra" + c.strId + Random.nextInt())
+    // }
     val interFlie = os.temp(dir = os.pwd, suffix = "jjj", deleteOnExit = true)
     val writer = new CatraWriter(interFlie.toString())
     writer.write(toCatraInput)
     writer.close()
     val arguments = CommandLineOptions(
       inputFiles = Seq(interFlie.toString()),
       timeout_ms = Some(OFlags.timeout),
-      trace = false,
-      printDecisions = false,
       dumpSMTDir = None,
       dumpGraphvizDir = None,
+      printDecisions = false,
       runMode = SolveSatisfy,
       backend = ChooseNuxmv,
       checkTermSat = true,
       checkIntermediateSat = true,
       eliminateQuantifiers = true,
       dumpEquationDir = None,
-      nrUnknownToMaterialiseProduct = 2,
```

Comment:
N.B: this only affects the lazy backend and does nothing for nuxmv!

Reply:
Good, looking forward to your release of Catra! It seems that we should discuss the integration more :)